### DISCUSSION TITLE
AutoHotkey dependency in Arduino IDE and nmap

### DISCUSSION THREAD

**Discussion by andi-blafasl** [chocolatey-community/chocolatey-packages#1827] [Ideas]
Change dependency for AutoHotkey from the explicit autohotkey.portable (Arduino) and autohotkey.install (nmap) to the Meta/virtual Package "AutoHotkey" so you don't end up with two separate AutoHotkey installations when installing both Packages.

**Comment by pauby** [comment]
The [arduino](https://community.chocolatey.org/packages/Arduino) package is not maintained here. Can you go back to the package page and use the 'Contact Maintainer' link?
